src/components/ui/Dialog.tsx

```diff
@@ -63,20 +63,30 @@ const DialogContent = forwardRef<
       {...props}
     >
       {children}
-      <DialogClose className={cn('absolute right-3 top-3')}>
+      {/* Move DialogClose to the header */}
+
+      {/* <DialogClose className={cn('')}>
         <Icon icon={CrossIcon} color="black" size={3.5} />
         <span className="sr-only">Close</span>
-      </DialogClose>
+      </DialogClose> */}
     </PrimitiveContent>
   </DialogPortal>
 ));
 DialogContent.displayName = PrimitiveContent.displayName;
 
 const DialogHeader = ({ className, ...props }: HTMLAttributes<HTMLDivElement>) => (
-  <div
-    className={cn('h-12 px-4 border-b border-gray-light', 'flex items-center', className)}
+  // Add a new parent div
```

Comment:
You do not need this kind of comment. It is more like a thing that should be added as a PR comment, not a direct comment in the code.